services/core-web/package.json

```diff
@@ -69,12 +69,29 @@
     "styled-components": "^5.2.0",
     "test-reducer-package": "^1.0.14",
     "tus-js-client": "^1.7.1",
-    "@syncfusion/ej2-react-filemanager": "^18.4.30",
-    "@syncfusion/ej2-base": "^18.4.31",
-    "@syncfusion/ej2-grids": "^18.4.33",
-    "@syncfusion/ej2-buttons": "^18.4.33",
-    "@syncfusion/ej2-layouts": "^18.4.30",
-    "@syncfusion/ej2-navigations": "^18.4.33",
+    "@syncfusion/ej2-react-filemanager": "^19.1.54",
+    "@syncfusion/ej2-react-pdfviewer": "^19.1.57",
+    "@syncfusion/ej2-react-documenteditor": "^19.1.57",
+    "@syncfusion/ej2-react-spreadsheet": "^19.1.57",
+    "@syncfusion/ej2-react-base": "^19.1.57",
+    "@syncfusion/ej2-pdfviewer": "^19.1.57",
+    "@syncfusion/ej2-documenteditor": "^19.1.57",
+    "@syncfusion/ej2-spreadsheet": "^19.1.57",
```

Comment:
This looks like a lot of packages being added, do we know how much it affects the size of our bundle?

Reply:
(Discussed on Teams)